.github/workflows/codeql-analysis.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       run: |
         mkdir -p $GITHUB_WORKSPACE/build
         cd $GITHUB_WORKSPACE/build
-        meson .. -Dman=false -Defi_binary=false -Dplugin_uefi_capsule_splash=false --prefix=$GITHUB_WORKSPACE/dist
+        meson setup .. -Dman=false -Defi_binary=false -Dplugin_uefi_capsule_splash=false --prefix=$GITHUB_WORKSPACE/dist
```

Comment:
While touching those lines can we perhaps also drop the efi binary and uedi capsule splash stuff? I would think meson features does that for us now.